src/js/tabs/motors.js

```diff
@@ -685,14 +687,17 @@ motors.initialize = async function (callback) {
         });
 
         const dshotBidirElement = $('input[id="dshotBidir"]');
+        const dshotEdtElement = $('input[id="dshotEdt"]');
```

Comment:
690, 697 and 700 should have it's own `if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_46) {` block

Reply:
done